### PR TITLE
Added Daily Bruin header to Stonewall

### PR DESCRIPTION
**Pull request by audreyliao**
Added header!  also ignore the stuff i commented out i thought i deleted it but perhaps not

### CODE REVIEW

File: pages/stonewall.jsx [hunk 1]
```diff
@@ -189,6 +253,7 @@ const Stonewall = () => {
                     that here, stone by stone. No stonewalling that week, no new
                     stone. Below, you can click each stone to read about why it's
                     there.
+
```
Comment (BrysonXiao) [inline pages/stonewall.jsx]: remove new line

File: pages/stonewall.jsx [hunk 2]
```diff
@@ -30,7 +30,71 @@ const Stonewall = () => {
...
+        //       <EmailPopUp
+        //         sub2Newsletter={this.subscribeToNewsletter}
+        //         close={this.closeNewsletterPopup}
+        //       />
+        //     ) : null}
+        //   </>
+        // );
+
     return (
+
```
Comment (BrysonXiao) [inline pages/stonewall.jsx]: remove new line

File: pages/stonewall.jsx [hunk 3]
```diff
@@ -30,7 +30,71 @@ const Stonewall = () => {
...
+        //     />
+        //     {this.state.showNewsletterPopUp ? (
+        //       <EmailPopUp
+        //         sub2Newsletter={this.subscribeToNewsletter}
+        //         close={this.closeNewsletterPopup}
+        //       />
+        //     ) : null}
+        //   </>
+        // );
+
```
Comment (BrysonXiao) [inline pages/stonewall.jsx]: remove these comments

**Review by BrysonXiao — CHANGES_REQUESTED** [review]
Can you remove the newlines and commented lines that you added? Usually we try to keep the file as "clean" as possible. To do this you can remove the lines i commented on, save the file, then git add, git commit, and git push again (make sure on the same branch). then i should be able to review it again and approve it :D

**Comment by audreyliao** [comment]
Ah my bad I totally forgot to delete that before meeting, I just changed it all and did git push again but the link doesn't take me to a new pull request let me know if I should try to do it again but as of now when I run git status it says everything's up to date

**Review by BrysonXiao — APPROVED** [review]

**Pull request merged**